OloEngine/src/OloEngine/Audio/SoundGraph/SoundGraphSound.cpp

```
@@ -350,23 +368,34 @@ namespace OloEngine::Audio::SoundGraph
+        // any resume the budget's own clock has not yet run past. It cannot satisfy a LATER
+        // position: for a procedural graph, getting to frame N means computing frames
+        // 0..N, so honouring the extra would spend exactly the CPU the suspension saved,
+        // in one audio callback. (The graph runtime is roughly real-time in a Debug build —
+        // docs/design/soundgraph-metasounds.md — so even a bounded catch-up burst is an
+        // underrun, not an optimisation. That is why there is no seek here rather than
+        // that nobody wrote one.)
+        //
+        // The engine self-corrects rather than carrying the lie: OnVoiceQueryPosition is
+        // authoritative while a voice is audible, so VoiceManager::Update pulls its record
+        // back to the graph's real frame on the very next tick.
+        (void)positionSeconds;
```

Cppcheck (2.21.0)

[style] The function 'SetInputDefault' is never used.

(unusedFunction)